jiwer/process.py

```diff
@@ -386,6 +399,21 @@ def _is_list_of_list_of_strings(x: Any, require_non_empty_lists: bool):
 
 
 def _word2char(reference: List[List[str]], hypothesis: List[List[str]]):
```

Comment:
nit: similarly, _word2int

Reply:
Sure! Done in https://github.com/jitsi/jiwer/pull/100/commits/0fe59c0d2c24f1588239acdc71a216e322c85a45